package.json

```diff
@@ -1,8 +1,7 @@
 {
-  "name": "opentype.js",
+  "name": "opentype.yne.js",
   "description": "OpenType font parser",
-  "version": "2.0.0",
-  "sideEffects": false,
+  "version": "1.3.3",
```

Comment:
@yne we're working on version 2.0.0 right now, 1.3.3 released a long time ago.

Reply:
its a mistake from me.
the version shall be left empty/null since it will be set by the github action upon release

Reply:
I changed to the original name `opentype.js`.
Beware that it will:
- break my repo when I create a tag (since I can't publish `opentype.js` on NPM)
- break this PR if we don't setup an `NPM_TOKEN`

Reply:
Did we add back in the ``"sideEffects": false,`` entry? It's apparently necessary for tree shaking.

Reply:
@ILOVEPIE 
it's a webpack-only specific flags (not ESBuild)